### PR TITLE
[script.module.tubed.api] v1.0.0

### PR DESCRIPTION
**Pull request by anxdpanic**
### Description
Required by https://github.com/xbmc/repo-plugins/pull/3233

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.module.tubed.api/resources/lib/src/httpd/handler.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+                    'SD': (1280 * 528) - 1,
+                    'HD720': 1280 * 720,
+                    'HD': 7680 * 4320
+                }
+
+                if 'HD' in authorized_types:
+                    size_limit = formats['HD']
+
+                elif 'HD720' in authorized_types:
+                    if xbmc.getCondVisibility('system.platform.android') == 1:
```
Comment (enen92) [inline script.module.tubed.api/resources/lib/src/httpd/handler.py:171]: We need to think about exposing this to the python api

**Review by enen92 — APPROVED** [review]
Thanks!

File: script.module.tubed.api/resources/lib/src/httpd/handler.py (hunk 1 above)
Comment (anxdpanic) [reply script.module.tubed.api/resources/lib/src/httpd/handler.py:171]: which part? This is YouTube specific widevine stuff

**Comment by anxdpanic** [comment]
You beat me to it by minutes heh, I'll have a quick follow up depending on testing, I forgot to enable API restrictions which is surely going to be required for the quota request. Should still be functional as is, as I can just remove the restriction until the new code is merged, which is a nice tiny change :)

**Pull request merged**